### ISSUE TITLE
Index file of suru icons inherit humanity, hicolor 

### ISSUE THREAD

**Issue by Paz-it** [snwh/suru-icon-theme#82]
That leads to ugly outdated icons in some apps : 
![screenshot from 2018-08-04 01-31-25](https://user-images.githubusercontent.com/37431503/43668611-88e7655e-9786-11e8-96a1-71cf4a8745e7.png)
![screenshot from 2018-08-04 01-32-42](https://user-images.githubusercontent.com/37431503/43668620-930dec88-9786-11e8-86f1-6fd9808590d7.png)
![screenshot from 2018-08-04 01-33-17](https://user-images.githubusercontent.com/37431503/43668629-9aa5922a-9786-11e8-976c-490c96406137.png)

I think that should be changed so the modern awesome suru icons will replace them. 
Thanks in advance for looking into it!

**Comment by snwh** [comment]
Yes, those icons do not exist in Suru and re-drawing them all for inclusion is a mammoth undertaking. Unless someone hires me to flesh out Suru with all the hundreds of legacy icons that there are, I'm not about to dedicate months of free time and effort required to execute on that task properly so Humanity (designed by people Canonical hired) as a fallback will have to do.

**Comment by Paz-it** [comment]
So sad! -- :woman_shrugging: 
Well, I'll wait for the finished development and then change the inherit thing at least in my machine :man_facepalming: 
Thanks for the quick reply, @snwh
